app/blog/2018-01-10-polymer-3-new-preview.md

```diff
@@ -0,0 +1,102 @@
+
+The 3.0 preview release of the Polymer core library includes two new features:
+
+* `PropertiesMixin`. This mixin breaks out a small subset of the lower Polymer layers responsible for defining declarative properties, creating accessors, and syncing attributes to properties. 
+
+  Users of lit-html and other templating alternatives may find `PropertiesMixin` useful. It provides a single `_propertiesChanged` callback and can be used to create new, lightweight base classes. 
+
+  To illustrate, the following example shows a very simple element using `PropertiesMixin`:
+
+  ```js
+  import {PropertiesMixin} from "../node_modules/@polymer/polymer/lib/mixins/properties-mixin.js"
+
+    class MyPropertiesElement extends PropertiesMixin(HTMLElement) {
+      // Define properties to watch. Only types are supported.
+      static get properties() {
+        return {
+          name: String
+        }
+        // Called whenever the declared properties change. 
+        _propertiesChanged(currentProps, changedProps, oldProps) {
+          // Render the changed content.
+          this.textContent = `Hello, ${this.name}`;
+        }
+      }
+    }
+  }
+  ```
+
+  For more information, see the source for `PropertiesMixin`. Documentation will be coming in a future release. 
+
+  For people interested in using lit-html, an experimental base class using `PropertiesMixin` with lit-html templating is available [somewhere](https://). 
+
+* `html` tag function. Starting with this preview, you need to return an instance of `HTMLTemplateElement` from your element’s static template getter—the option to return a string has been removed. This change makes the template type predictable, which in turn will make it simpler to extend superclass templates.
+
+  The easiest way to return an `HTMLTemplateElement` from your template getter is to use Polymer’s new `html` helper, which automatically generates an `HTMLTemplateElement` from a template literal string, as shown here: 
+
+  ```js
+  import {Element as PolymerElement, html} from "../node_modules/@polymer/polymer/polymer-element.js"
+  class MyAppElement extends PolymerElement {
+    static get template() {
+      return html`<div>I'm a template</div>
+                  <div>[[withBindings]]</div>
+                  <button on-click="clickHandler">Click me!</button>`
+    }
+    ...
+  }
+  customElements.define('my-app-element', MyAppElement);
+  ```
+
+These new features will also be released in the next minor release of 2.x. However, there's no need to manually convert your 2.x code to use the `html` helper—the Polymer Modulizer will make that conversion for you.
+
+Other than these changes, APIs haven't changed, and you can follow the instructions in our previous blog post, [Hands-on with the Polymer 3.0 preview](2017-08-23-hands-on-30-preview).
+
+So give the latest Polymer 3.0 preview a try. Send us feedback, report bugs, and let us know what you think!
```

Comment:
## What's coming next?

For an idea of what's coming down the pike, keep an eye on the roadmap and element status info we've added to the `polymer-modulizer` README.

As we said above, you can expect regular preview releases for the duration of the Polymer 3.0 project—look for the next one to land the week of January 22.

Based on feedback from early adopters of 3.0, we'll be focusing next on smoothing out some rough edges around package installation and module import specifiers. Watch this space for details!